lib/ddtrace/utils/mass_tagger.rb

```diff
@@ -0,0 +1,59 @@
+module Datadog
+  module Utils
+    # Helper class to used to tag configured headers
+    module MassTagger
```

Comment:
`Datadog::Utils::MassTagger` is a bit ambiguous. What kind of object is it tagging? What kind of data is it doing en masse?

I'd suggest making the name more specific, or splitting the behavior into smaller modules/files that are more specific. Perhaps something like: `Datadog::Utils::Tagging::Rack::Request` or `Datadog::Utils::Tagging::Rack::Response`?

Furthermore... is this the best place for this behavior? I have mixed feelings that something named "Rack" exists in the main library. Rack is a very specific integration, and I think our main library probably should reference a minimal number of external integrations, even if its only doing so by name.

In this sense, my suggestion here would to move this behavior under `ddtrace/contrib/rack/utils` namespace. Any other integration that requires this behavior, e.g. Rails or Sinatra which specifies Rack as a dependency anyways, could then `require 'ddtrace/contrib/rack/utils/tagging/request` or `require 'ddtrace/contrib/rack/utils/tagging/response`.

Reply:
I've renamed MassTagger to simply `Tagger`. As its quite generic and could be used outside of Rack (i.e. GRPC).

Rack specific code now lives under `contrib/rack/tagging`. Thanks for suggesting it, I wasn't sure if code should be shared among 'contribs'.